cmd/kube-proxy/app/server_others.go

```diff
@@ -329,6 +336,49 @@ func (s *ProxyServer) setupConntrack() error {
 		}
 	}
 
+	if s.Config.Conntrack.UDPTimeout.Duration > 0 {
+		timeout := int(s.Config.Conntrack.UDPTimeout.Duration / time.Second)
+		if err := ct.SetUDPTimeout(timeout); err != nil {
+			return err
+		}
+	}
+
+	if s.Config.Conntrack.UDPStreamTimeout.Duration > 0 {
+		timeout := int(s.Config.Conntrack.UDPStreamTimeout.Duration / time.Second)
+		if err := ct.SetUDPStreamTimeout(timeout); err != nil {
+			return err
+		}
+	}
+
+	if s.Config.Conntrack.TCPBeLiberal != nil {
+		if *s.Config.Conntrack.TCPBeLiberal {
+			if err := ct.SetTCPBeLiberal(1); err != nil {
+				return err
+			}
+		}
+	} else {
+		// try setting nf_conntrack_tcp_be_liberal=1 if kernelVersion < CTStateInvalidFixKernelVersion
+		// don't return any error, just log them.
+		kernelVersionStr, err := ipvs.NewLinuxKernelHandler().GetKernelVersion()
```

Comment:
Maybe we should move `LinuxKernelHandler` out of ipvs proxier, kubelet is also using it.

Reply:
(filed #120895)